### DISCUSSION TITLE
添加一个sdk后报错

### DISCUSSION THREAD

**Discussion by H69220818** [midwayjs/midway#3789] [Q&A]
### Describe the problem（描述问题）

start error: Attempt to change mapping for "form" extension from "application/x-www-form-urlencoded" to "application/x-www-form-urlencoded". Pass `force=true` to allow this, otherwise remove "form" from the list of extensions for "application/x-www-form-urlencoded".
Error: Attempt to change mapping for "form" extension from "application/x-www-form-urlencoded" to "application/x-www-form-urlencoded". Pass `force=true` to allow this, otherwise remove "form" from the list of extensions for "application/x-www-form-urlencoded".

这是报错内容

### Midway Versions（Midway 版本）

midway-version@1.3.8

**Comment by czy88840616** [comment]
堆栈是啥
